packages/table/src/Table.ts

```diff
@@ -507,6 +507,11 @@ export class Table extends SizedMixin(SpectrumElement, {
                 }
             );
         }
+        if (this.items.length) {
+            // Ensures screenreaders can announce the true size of the table
+            // despite virtualization only rendering a subset.
+            this.setAttribute('aria-rowcount', `${this.items.length}`);
+        }
```

Comment:
This method is gated by this test, so we might not need it here.

Also, do we want to remove this attribute ever? E.G. we wouldn't come into this method is `this.items` was set to `[]` later on, but should `aria-rowcount="0"` in that case, or be removed?

Reply:
You're right, this conditional isn't necessary here!